code/game/machinery/traitorous_cloner.dm

```diff
@@ -0,0 +1,355 @@
+//Experimental cloner; clones a body regardless of the owner's status, letting a ghost control it instead
+
+/obj/machinery/clonepod/traitorous
```

Comment:
Why did you copy/paste the experimental cloner code instead of just making this a subtype, or even just making experimental cloner emaggable?

Reply:
Cuz then it will actually be experimental cloning pod traitorous...

Reply:
?

Reply:
Done.